tests/testsuite/install_upgrade.rs

```diff
@@ -799,11 +799,8 @@ fn already_installed_updates_yank_status_on_upgrade() {
 
     cargo_process("install foo --version=1.0.1")
         .with_status(101)
-        .with_stderr(
-            "\
-[UPDATING] `[..]` index
-[ERROR] could not find `foo` in registry `[..]` with version `=1.0.1`
-",
+        .with_stderr_contains(
+            "error: cannot install package `foo`, it has been yanked from registry `https://github.com/rust-lang/crates.io-index`",
```

Comment:
This string could be split up, too.

Reply:
Thanks for the suggestion. I've updated the PR, please have a look.